medusa/providers/generic_provider.py

```diff
@@ -829,14 +829,25 @@ def add_cookies_from_ui(self):
             return {'result': False,
                     'message': 'Cookie is not correctly formatted: {0}'.format(self.cookies)}
 
-        if not all(req_cookie in [x.rsplit('=', 1)[0] for x in self.cookies.split(';')]
-                   for req_cookie in self.required_cookies):
-            return {
-                'result': False,
-                'message': "You haven't configured the requied cookies. Please login at {provider_url}, "
-                           'and make sure you have copied the following cookies: {required_cookies!r}'
-                           .format(provider_url=self.name, required_cookies=self.required_cookies)
-            }
+        if self.required_cookies:
+            if self.name != 'Beyond-HD':
```

Comment:
For now beyond-hd is the only one with a cookie name like: remember_web_59ba36addc2b2f9401580f01445acd8ea4e30af9d. So I had to make an exception for this provider.